src/super_gradients/training/models/model_registry.py

```diff
@@ -0,0 +1,15 @@
+ARCHITECTURES = {}
+KD_ARCHITECTURES = {}
+
+
+def register(fn):
```

Comment:
1. so are you ok with changing all the function names in the code so it would fit the original naming?
```
@register
def ResNet18Cifar(arch_params, num_classes=None):  # this will be changed to resnet18_cifar
    return CifarResNet(BasicBlock, [2, 2, 2, 2], num_classes=num_classes or arch_params.num_classes)
```
2.  In `register(fn)` it is a fn, not a cls; it is a fn that returns a cls

Reply:
@shaydeci suggested that `registry` will get a parameter so you can override the default naming (i.e. the function's name)
`def register(key=None)`